README.md

```diff
@@ -1,45 +1,99 @@
+# Sherlock Platform
+
+## About
+
+This is the customized IntelliJ platform for Sherlock Project - an IntelliJ-based implementation of the new GPU Profiler tool. This is based
+off the IntelliJ Platform Version 2.1. To read more on the IntelliJ Platform documentation scroll down to the IntelliJ IDEA Community
+Edition section.
+
+## Developer Documentation
+
+### Copyright for Source Code Headers
+
+Every file containing source code must include copyright and license information.
+Make sure you add the following header when creating a new source file:
+
+```code
+Copyright 2024 Google LLC
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+https://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+```
+
+Incremental updates to existing source files should update the year.
+
+### Build Instructions for Sherlock Platform
+
+* Run `chmod +x ./build_sherlock_platform.sh` from the root of the directory to grant access.
+* Run `./build_sherlock_platform.sh` to build the platform.
+* Extract the artifact from `out/sherlock-platform/artifacts` according to the OS.
+* Run the platform from `out/sherlock-platform/artifacts/sherlock-platform-242.21829/Sherlock\ Platform-2024.2.1/bin/`
+
 # IntelliJ IDEA Community Edition [![official JetBrains project](http://jb.gg/badges/official.svg)](https://github.com/JetBrains/.github/blob/main/profile/README.md)
-These instructions will help you build IntelliJ IDEA Community Edition from source code, which is the basis for IntelliJ Platform development.
+
```

Comment:
Could you revert the whitespace/endline changes in the remainder of this file?

Reply:
Done. Intellij reformated the file when I did "Reformat Code"

Reply:
What I would do:

```
git reset HEAD^ --soft
git restore --staged README.md
git difftool
git add README.md
git commit   # copy paste commit description from PR
git push --force
```

Reply:
Or, get the original `README.md` from main branch, copy it into your feature branch onto your existing commit, and then use `git diff` to replay your changes and ignore those whitespaces.

Reply:
Done.